assets/src/stories-editor/blocks/amp-story-cta/edit.js

```diff
@@ -94,7 +95,17 @@ const CallToActionEdit = ( {
 					<RichText
 						placeholder={ placeholder }
 						value={ text }
-						onChange={ ( value ) => setAttributes( { text: value } ) }
+						onChange={ ( value ) => {
+							setAttributes( { text: value } );
+							// Also update width and height based on the room that the CTA button takes.
+							const element = document.querySelector( `#amp-story-cta-button-${ clientId } .amp-block-story-cta__link` );
```

Comment:
This seems to be the element from the `edit` component, which can have different dimensions than the saved block. Example:

When editing:

<img width="408" alt="Screenshot 2019-10-07 at 14 50 31" src="https://user-images.githubusercontent.com/841956/66339805-e360d280-e911-11e9-8f53-1b89d262e210.png">

Not editing:

<img width="449" alt="Screenshot 2019-10-07 at 14 50 34" src="https://user-images.githubusercontent.com/841956/66339809-e6f45980-e911-11e9-8130-b9fd54826590.png">

Frontend:

<img width="482" alt="Screenshot 2019-10-07 at 14 50 39" src="https://user-images.githubusercontent.com/841956/66339820-ea87e080-e911-11e9-9e08-db9e6dc4c01b.png">

Reply:
Thank you for testing. Looks like there was a CSS rule overwritten that made the edit and view look different for the CTA block, now it should look the same.

However, will check into amp-fit-text anyway so that might change things a bit again.

Reply:
I am still seeing issues around editing and previewing. 

![Screenshot from 2019-10-09 11-49-26](https://user-images.githubusercontent.com/237508/66475364-18446500-ea8b-11e9-9681-631e01fd6ffb.png)
![Screenshot from 2019-10-09 11-50-10](https://user-images.githubusercontent.com/237508/66475374-1bd7ec00-ea8b-11e9-8365-a8f8032689c0.png)

Reply:
Thanks for testing! Just in case: did you build the CSS before testing? There were some FE changes that don't get built automatically.

Line height was different indeed, adjusted that.

Reply:
CSS build fixed the fix for me, thanks for the tip.

Shouldn't the button have a max width of the container that force to the next line. I can't see a good reason why you would want a button that runs on forever like that. To make text go onto a new line, , I had to press enter, which left really weird.

Reply:
Maybe, that's out of scope for this bug though. We generally allow (at this point) the blocks to be "over the edge" so these two things might need some UX thought together. E.g. if the CTA button is in the middle and you start typing and we allow 100% width then it can still go over the edge.

Quite certain it's a lower priority compared to some other issues.